dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -349,6 +358,42 @@ class CompileTest {
 
     return _kSdkNameToMetricNameMapping[sdkName];
   }
+
+  static Future<Map<String, dynamic>> getSizesFromApk(String apkPath) async {
+    final  String output = await eval('unzip', <String>['-v', apkPath]);
+    final List<String> lines = output.split('\n');
+
+    final Map<String, _UnzipListEntry> fileToMetadata = <String, _UnzipListEntry>{};
+
+    // First three lines are header, last two lines are footer.
+    for (int i = 3; i < lines.length - 2; i++) {
+      final List<String> line = lines[i].trim().split(new RegExp('\\s+'));
+      assert(line.length == 8);
+      fileToMetadata[line[7]] = new _UnzipListEntry(line);
+    }
+
+    final _UnzipListEntry icudtl = fileToMetadata['assets/icudtl.dat'];
+    final _UnzipListEntry libflutter = fileToMetadata['lib/armeabi-v7a/libflutter.so'];
+    final _UnzipListEntry isolateSnapshotData = fileToMetadata['assets/isolate_snapshot_data'];
+    final _UnzipListEntry isolateSnapshotInstr = fileToMetadata['assets/isolate_snapshot_instr'];
+    final _UnzipListEntry vmSnapshotData = fileToMetadata['assets/vm_snapshot_data'];
+    final _UnzipListEntry vmSnapshotInstr = fileToMetadata['assets/vm_snapshot_instr'];
+
+    return <String, dynamic>{
```

Comment:
I wonder if the dashboard UI is ready to onboard so many new metrics? :smile:

Reply:
Hehe. I actually cleaned up the dashboard on Friday and removed all deprecated (dart1) benchmarks.